### PR TITLE
Basics for cli

### PR DESCRIPTION
**Pull request by justinvdm**
The plan is to make something like this possible for now:

```
$ numl json dialogue.numl
```

Once we have an api to send things to, we can extend this to do something like:

```
$ numl put dialogue.numl
```

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #5, otherwise ready for review.

File: src/cli/json.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+var numl = require('../../numl');
+
+
+function run(argv) {
+  try {
+    log.out(JSON.stringify(numl(open(argv.file).toString())));
+  }
+  catch(e) {
+    log.err(e);
+  }
```
Comment (hodgestar) [inline src/cli/json.js:13]: Should we be returning some sort of exit code here?

File: package.json [hunk 2]
```diff
@@ -21,10 +24,15 @@
...
     "pegjs-loader": "^0.4.0",
+    "shelljs": "^0.6.0",
     "webpack-stream": "^3.1.0"
   },
   "dependencies": {
+    "chalk": "^1.1.1",
+    "dedent": "^0.6.0",
     "lodash": "^4.6.0",
-    "to-camel-case": "^1.0.0"
+    "log-syntax-error": "^0.1.1",
```
Comment (hodgestar) [inline package.json]: Is this dependency still being used somehow?

**Comment by hodgestar** [comment]
Two questions, otherwise looks good.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**